packages/taquito/test/subscribe/observable-subscription.spec.ts

```diff
@@ -32,7 +31,7 @@ describe('Observable subscription test', () => {
         flush();
 
         expect(stub).toBeCalledTimes(3);
-        expect(errStub).toBeCalledTimes(2);
```

Comment:
From my understanding, the stub is called 3 times (a,b,c), and then an error happens (#). 
It seems that nothing happens after the error (d-e are never called.. I would have expected the stub to have been called 5 times with this change). This test was to make sure that the observable retries on error when the property "shouldRetry" is set to true, but it seems that the retry is not tested.

Reply:
This is fixed.